styles/themes/teammember/normalize.scss

```diff
@@ -0,0 +1,186 @@
+/* http://meyerweb.com/eric/tools/css/reset/  v2.0b1 | 201101  */
```

Comment:
This is the one that's being used in the Workwise login page and seems that it's a different set of rules but you are right that we should use one or the other